### PR TITLE
[Repository] Removed unused ``createFilterPaginator`` methods

### PR DESCRIPTION
**Pull request by Zales0123**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Related tickets |
| License         | MIT

### CODE REVIEW

File: src/Sylius/Component/Core/Repository/OrderRepositoryInterface.php [hunk 1]
```diff
@@ -91,14 +91,6 @@ public function findOneForPayment($id);
...
      */
-    public function createFilterPaginator(array $criteria = null, array $sorting = null);
-
-    /**
-     * @param array $criteria
-     * @param array $sorting
-     *
-     * @return PagerfantaInterface
-     */
     public function createCheckoutsPaginator(array $criteria = null, array $sorting = null);
```
Comment (michalmarcinkowski) [inline src/Sylius/Component/Core/Repository/OrderRepositoryInterface.php]: How about other `...Paginator` methods? Are they used anywhere? If not we should remove them also.

File: src/Sylius/Bundle/CoreBundle/Doctrine/ORM/ProductRepository.php [hunk 2]
```diff
@@ -80,43 +80,6 @@ public function createByTaxonAndChannelPaginator(TaxonInterface $taxon, ChannelI
...
-
-        $this->applySorting($queryBuilder, $sorting);
-
-        return $this->getPaginator($queryBuilder);
-    }
-
-    /**
-     * {@inheritdoc}
-     */
     public function findForDetailsPage($id)
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/CoreBundle/Doctrine/ORM/ProductRepository.php]: Such methods are also remains from WebBundle, please look through the codebase to catch as much unused repository methods as possible ;)

File: src/Sylius/Component/Core/Repository/ProductRepositoryInterface.php [hunk 3]
```diff
@@ -39,15 +39,6 @@ public function createByTaxonPaginator(TaxonInterface $taxon, array $criteria =
      * @return PagerfantaInterface
      */
     public function createByTaxonAndChannelPaginator(TaxonInterface $taxon, ChannelInterface $channel);
```
Comment (michalmarcinkowski) [inline src/Sylius/Component/Core/Repository/ProductRepositoryInterface.php]: Should be removed also :)

File: src/Sylius/Bundle/CoreBundle/Doctrine/ORM/ProductRepository.php (hunk 2 above)
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/CoreBundle/Doctrine/ORM/ProductRepository.php]: Ping :)

**Comment by michalmarcinkowski** [comment]
@Zales0123 rebase please :)

**Comment by michalmarcinkowski** [comment]
👍

**Comment by emodric** [comment]
Hi!

What methods should be used instead of `ProductRepository::createByTaxonPaginator` and `ProductRepository::createFilterPaginator` ?

**Pull request merged**